configs/xcache-consistency-check/xrootd/xcache-consistency-check.cfg

```diff
@@ -17,3 +17,9 @@ num_procs = 1
 # Once a file has been analyzed it will wait for this many seconds
 # before analyzing it again, even if the file has changed
 last_check_threshold = 86400
+
+# Comma separated list of directories where the tool is allowed to remove from
+# 'auto' will add the paths related to the directives 'localroot' and 'pfc.spaces'
+# within the xrootd config file
+# If not set, the tool is allowed to remove corrupt file from any directory
+remove_only_from = auto
```

Comment:
Does this mean it will also remove the meta information from the meta partition

Reply:
Otherwise ooks good to me

Reply:
@ddavila0

Reply:
yes, it looks for the pfc.spaces and gets both labels e.g. 'meta' and 'data', then it looks for all oss.space directives with either 'meta' or 'data' on it and copies the path on them. This way the tool is allowed to remove both .root and .cinfo files